react-app/src/app/screens/Game/components/Moves/components/Move/index.js

```diff
@@ -1,33 +1,32 @@
-import React, { Component, Fragment } from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import { jumpTo } from '../../../../../redux/Game/actions';
-import calculateWinner from '../../utils';
+import { jumpTo } from '../../../../../../../redux/Game/actions';
+import calculateWinner from '../../../../utils';
 
 import style from './styles.scss';
 
 class Move extends Component {
-  createHistoryButton(move) {
-    const desc = move ? `Go to move # ${move}` : 'Go to game start';
-    const history = this.props.history.slice(0, move + 1);
+  render() {
+    const desc = this.props.move ? `Go to move # ${this.props.move}` : 'Go to game start';
+    const history = this.props.history.slice(0, this.props.move + 1);
     const current = history[history.length - 1];
     const squares = current.squares.asMutable().slice();
```

Comment:
I think it would be great to map desc and squares in the local state, in order to avoid recalculating them in each render. You can set them in the `getDerivedStateFromProps` method.
https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops